dist/constants.json

```diff
@@ -7825,697 +7825,697 @@
       },
       {
         "value": "516GH",
-        "label": "SEBRING VA CLINIC"
+        "label": "Sebring VA Clinic"
       },
       {
         "value": "520BZ",
-        "label": "PENSACOLA VA CLINIC"
+        "label": "Pensacola VA Clinic"
       },
       {
         "value": "520GB",
-        "label": "PANAMA CITY BEACH VA CLINIC"
+        "label": "Panama City Beach VA Clinic"
       },
       {
         "value": "546GD",
-        "label": "PEMBROKE PINES VA CLINIC"
+        "label": "Pembroke Pines VA Clinic"
       },
       {
         "value": "546GE",
-        "label": "KEY LARGO VA CLINIC"
+        "label": "Key Largo VA Clinic"
       },
       {
         "value": "548GA",
-        "label": "FORT PIERCE VA CLINIC"
+        "label": "Fort Pierce VA Clinic"
       },
       {
         "value": "548GB",
-        "label": "DELRAY BEACH VA CLINIC"
+        "label": "Delray Beach VA Clinic"
       },
       {
         "value": "548GC",
-        "label": "STUART VA CLINIC"
+        "label": "Stuart VA Clinic"
       },
       {
         "value": "548GD",
-        "label": "BOCA RATON CBOC"
+        "label": "Boca Raton VA Clinic"
       },
       {
         "value": "548GE",
-        "label": "VERO BEACH VA CLINIC"
+        "label": "Vero Beach VA Clinic"
       },
       {
         "value": "548GF",
-        "label": "OKEECHOBEE VA CLINIC"
+        "label": "Okeechobee VA Clinic"
       },
       {
         "value": "573GI",
-        "label": "THE VILLAGES VA CLINIC"
+        "label": "The Villages VA Clinic"
       },
       {
         "value": "675GA",
-        "label": "VIERA VA CLINIC"
+        "label": "Viera VA Clinic"
       },
       {
         "value": "675GE",
-        "label": "TAVARES VA CLINIC"
+        "label": "Tavares VA Clinic"
       },
       {
         "value": "675GF",
-        "label": "CLERMONT CBOC"
+        "label": "Clermont VA Clinic"
       },
       {
         "value": "520GC",
-        "label": "EGLIN AIR FORCE BASE VA CLINIC"
+        "label": "Eglin Air Force Base VA Clinic"
       },
       {
         "value": "573GK",
-        "label": "MARIANNA VA CLINIC"
+        "label": "Marianna VA Clinic"
       },
       {
         "value": "675GD",
-        "label": "DELTONA VA CLINIC"
+        "label": "Deltona VA Clinic"
       },
       {
         "value": "675GB",
-        "label": "WILLIAM V. CHAPPELL, JR. VOC"
+        "label": "William V. Chappell, Jr. Veterans' Outpatient Clinic"
       },
       {
         "value": "675GC",
-        "label": "KISSIMMEE VA CLINIC"
+        "label": "Kissimmee VA Clinic"
       },
       {
         "value": "675GG",
-        "label": "LAKE BALDWIN VA CLINIC"
+        "label": "Lake Baldwin VA Clinic"
       },
       {
         "value": "573GL",
-        "label": "PALATKA VA CLINIC"
+        "label": "Palatka VA Clinic"
       },
       {
         "value": "673BZ",
-        "label": "PORT RICHEY OUTPATIENT CLINIC"
+        "label": "New Port Richey VA Clinic"
```

Comment:
Let's confirm this name change.

Port Richy and New Port Richy are two different cities in FL.